### PR TITLE
change seealso to seealsoContent

### PR DESCRIPTION
**Pull request by bianliu1013**
to fix: https://github.com/ericholscher/sphinx-docfx-yaml/issues/9

### CODE REVIEW

File: docfx_yaml/extension.py [hunk 1]
```diff
@@ -354,7 +354,7 @@ def build_finished(app, exception):
                         obj['summary'] = obj['syntax'].pop('summary')
                     # Raise up seealso
                     if 'seealso' in obj['syntax'] and obj['syntax']['seealso']:
-                        obj['seealso'] = obj['syntax'].pop('seealso')
+                        obj['seealsoContent'] = obj['syntax'].pop('seealso')
```
Comment (hellosnow) [inline docfx_yaml/extension.py:357]: >seealsoContent [](start = 29, length = 14)

also fix the test tests/test_yaml.py? #Closed

File: docfx_yaml/extension.py [hunk 2]
```diff
@@ -322,6 +322,10 @@ def build_finished(app, exception):
                 # Skip objects without a module
                 continue
 
+            if filename.lower().find('.tests') != -1:
```
Comment (hellosnow) [inline docfx_yaml/extension.py]: how about put it in another separate PR?

Comment (bianliu1013) [reply docfx_yaml/extension.py]: removed, will create another PR.

**Comment by hellosnow** [comment]
:shipit:

**Comment by bianliu1013** [comment]
https://github.com/ericholscher/sphinx-docfx-yaml/commit/983a58ea5e4c815a9b25006334dda5b72599c40f
already merged to master branch by eric. so close here.
